kiali-server/templates/service.yaml

```diff
@@ -34,11 +34,17 @@ spec:
   {{- end }}
     protocol: TCP
     port: {{ .Values.server.port }}
+    {{- if eq .Values.deployment.service_type "NodePort" }}
+    nodePort: {{ .Values.server.nodePort }}
+    {{- end }}
   {{- if .Values.server.observability.metrics.enabled }}
   - name: http-metrics
     appProtocol: http
     protocol: TCP
     port: {{ .Values.server.observability.metrics.port }}
+    {{- if eq .Values.deployment.service_type "NodePort" }}
+    nodePort: {{ .Values.server.observability.metrics.nodePort }}
+    {{- end }}
```

Comment:
I don't think we want or need to expose the metrics over the nodePort. For one thing, it isn't used - the Prometheus scrape endpoint is still defined as using the service port as you see [here](https://github.com/kiali/helm-charts/blob/v1.80.0/kiali-server/templates/deployment.yaml#L31). Secondly, I don't see the need to expose the metrics endpoint over the nodePort - it is only to be used by Prometheus when scraping the metrics, and Prometheus should be able to do that over the normal port.

So I believe this should be removed and we should only expose the HTTP endpoint over the nodePort.

 ```suggestion
```